java/client/src/main/java/org/apache/rocketmq/client/java/example/ProducerTransactionMessageExample.java

```diff
@@ -92,7 +70,5 @@ public static void main(String[] args) throws ClientException, IOException {
         transaction.commit();
         // Or rollback the transaction.
         // transaction.rollback();
-        // Close the producer when you don't need it anymore.
```

Comment:
it's necessary to reserve the shutdown annotation to notice adding producer shutdown to shutdown hook